### PR TITLE
feat: Advanced Windows features

### PR DESCRIPTION
**Pull request by pedrofaria**

### CODE REVIEW

**Comment by pedrofaria** [comment]
[accweb_dev.zip](https://github.com/assetto-corsa-web/accweb/files/9035443/accweb_dev.zip)

For testing purpose!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.19.0
+* Fix some typo errors. #210 #212
+* Fix float point handling for rain and cloud. #211
+* Introducing the advance windows features. #213
```
Comment (matslindh) [inline CHANGELOG.md:6]: Details?

File: internal/app/handler_servers.go [hunk 2]
```diff
@@ -79,3 +80,19 @@ func (h *Handler) StopAllServers(c *gin.Context) {
 
 	c.JSON(http.StatusOK, nil)
 }
+
+// Metadata Show some useful informations
```
Comment (matslindh) [inline internal/app/handler_servers.go:84]: This doesn't say anything useful

**Review by matslindh — COMMENTED** [review]
Can't say anything about the windows features, but these two are too general - doesn't say anything useful

**Pull request merged**